modules/utxo-ord/src/psbt.ts

```diff
@@ -115,8 +120,11 @@ export function findOutputLayoutForWalletUnspents(
     maxInscriptionOutput = BigInt(20_000),
   } = constraints;
 
-  const [input] = inputs;
-  const inscriptionOutput = new OrdOutput(input.value, [toSatRange(satPoint)]);
+  // Join all the inputs into a single inscriptionOutput.
+  // For the purposes of finding a layout there is no difference.
+  const inscriptionOutput = OrdOutput.joinAll(
+    inputs.map((i) => new OrdOutput(i.value, i === inputs[0] ? [toSatRange(satPoint)] : []))
```

Comment:
Hmm, this assumes that the order of inputs is set by the caller. kinda seems like we'd want to do something more like having the inscribed input in one param and then the other available inputs in a separate param so they can be added where needed to make reasonable sized fee and change outs.

Reply:
fixed

Reply:
* new argument `supplementalUnspents: WalletUnspent[]`
* new argument `minimizeInputs`